modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -42,3 +45,9 @@ topk(10,count by (job)({__name__=~".+"}))
 ** *If the metrics relate to a user-defined project*, review the metrics key-value pairs assigned to your workload. These are implemented through Prometheus client libraries at the application level. Try to limit the number of unbound attributes referenced in your labels.
 
 ** *If the metrics relate to a core {product-title} project*, create a Red Hat support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
+
+. Check the TSDB status in the Prometheus UI.
+.. In the *Administrator* perspective, navigate to *Networking* -> *Routes*.
+.. Select the `openshift-monitoring` project in the *Project:* list.
+.. Select the URL in the `prometheus-k8s` row to open the login page for the Prometheus UI.
```

Comment:
change `Select`  to `Click` is better

Reply:
> change to following is better
> Click the URL in the `prometheus-k8s` row to open the login page for the Prometheus UI and login.

Thank you for the wording suggestion. For consistency, I pulled these steps and wording from another procedure in the Monitoring docs for accessing third-party UIs: https://docs.openshift.com/container-platform/4.7/monitoring/accessing-third-party-uis.html#accessing-third-party-uis-using-the-web-console_accessing-third-party-uis